variables.tf.json

```diff
@@ -75,6 +75,13 @@
                 "type": "string"
             }
         },
+        {
+            "permissions_boundary_arn": {
+                "default": "",
+                "description": "ARN of the permissions boundary to be set on Clumio Roles.",
+                "type": "string"
+            }
+        },
```

Comment:
Curious why did this move? This is because before you had placed this variable manually and now it's getting generated?